tools/samtools_view.cwl

```diff
@@ -0,0 +1,89 @@
+class: CommandLineTool
+cwlVersion: v1.2
+id: samtools_view
+doc: |-
+  views and converts SAM/BAM/CRAM files
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/samtools:1.15.1'
+  - class: ResourceRequirement
+    ramMin: $(inputs.ram*1000)
+    coresMin: $(inputs.cpu)
+baseCommand: []
```

Comment:
Is it necessary to have _baseCommand_ if we do not have command line to run ?

Reply:
By spec definition, it is not required. I just keep it around for completeness.